src/components/scenes/ChangePasswordScene.tsx

```diff
@@ -157,7 +157,11 @@ const ChangePasswordSceneComponent = ({
             placeholder={lstrings.confirm_password}
             autoFocus={focusSecond}
             onChangeText={handleChangeConfirmPassword}
-            onSubmitEditing={handlePress}
+            onSubmitEditing={
+              confirmPassword !== '' && confirmPassword !== password
+                ? handlePress
+                : undefined
+            }
```

Comment:
If we hit the "enter" button with an incorrect value, nothing will happen. That seems like it would be more broken than whatever was going on before.